source/includes/rars_unique_domain.md

```diff
@@ -0,0 +1,46 @@
+## Unique Domain
+
+Returns a list of unique domains that are being tracked for a given advertiser in the past 14 days. The data returned is a list of unique domains along with corresponding visit counts.
+
+### Resource Overview
+
+| Method | URI Format |
+|---|---|
+| GET | /client_reports/unique-domain/[gmaid] |
```

Comment:
This is incorrect, it should be `unique_domains`, see [this](https://github.com/GannettDigital/reach-analytics-reporting-service/pull/1722/files#diff-959bc9abc46a55332bb64d5155a79323afa75a50ec1a2137ddd22d926f62c6c5R56)